aggregators/rules/ipg.js

```diff
@@ -1,17 +1,49 @@
+const jsDOM = require("jsdom");
 class IPG {
     constructor() {
-        this.Location = "http://blogs.magicjudges.org/rules/ipg/";
+        this.location = "http://blogs.magicjudges.org/rules/ipg";
     }
 
     find(parameter, callback) {
-        //todo
+        jsDOM.env(
+            this.location + parameter,
+            ["http://code.jquery.com/jquery.js"],
+            /**
+             *
+             * @param err
+             * @param window
+             * @param window.$ jquery
+             * @param window.$.each jquery for each loop
+             * @param window.$.prop jquery property
+             */
+            function(err,window){
+                let resultLength = (window.$(".entry-header h1:first").text()).length+6;
+                const result = ["**"+window.$(".entry-header h1:first").text()+"**\n"];
+                window.$(".alert-info").remove();
+                window.$(".entry-content h2, .entry-content p, .entry-content li, .entry-content td").each(function(){
+                    let text = window.$(this).text()+"\n";
```

Comment:
`<div class="alert alert-warning" role="alert">` should be handled separately, as it will otherwise should up between headline and description:
"A player commits one or more of the following errors involving his or her deck, decklist, or sideboard:
Penalty
Game Loss
The deck, decklist,..."